### PR TITLE
smithsonianmag.com #24278

### PR DESCRIPTION
**Pull request by slavaleleka**
.

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Use ext-CSS rule to hide the ad text in sidebar. Class name is unsafe.

Also please add the proposed rule by user.

**Comment by slavaleleka** [comment]
.sticky-block disappeared, I guess with such class name it's safe now. or it is not?

**Comment by BlazDT** [comment]
Yes, it is safe now. But the "||logs ..." rule must be moved to SpywareFilter/tracking_servers_firstparty.txt

File: SpywareFilter/sections/tracking_servers_firstparty.txt [hunk 1]
```diff
@@ -529,6 +529,7 @@
 ||d10lpsik1i8c69.cloudfront.net^$third-party
 ||stg-data-collector.playbuzz.com^
 ||stats.walytics.com^$third-party
+||logs*.smithsonian.museum^$domain=smithsonianmag.com
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers_firstparty.txt]: specific.txt

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -892,6 +892,7 @@ androidmtk.com###fbhide[style^="width:300px;height:600px;"]
 androidmtk.com###relatedpost
 androidmtk.com##.in-article-adsense
 slice.ca###ad-DoubleBigBox
+smithsonianmag.com##.main-wrapper > .sidebar > .special-offers-from-advertisers-ad-showcase
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: можно без `.main-wrapper > .sidebar >`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**